slack-plugin/build.gradle.kts

```diff
@@ -5,6 +5,10 @@ plugins {
   alias(libs.plugins.bestPracticesPlugin)
 }
 
+kotlinDslPluginOptions {
+  jvmTarget.set("11")
```

Comment:
Found out the hard way that kotlin-dsl overrides your jvm target too unless you set this 🙃